2019-12-16 17:36:20,091 ERROR 6435 [-/::1/-/7ms GET /] nodejs.TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    at JSON.stringify (<anonymous>)
    at respond (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:259:15)
    at handleResponse (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:164:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

pid: 6435
hostname: C02Y81Q2JG5L

2019-12-16 17:36:41,618 ERROR 6549 [-/::1/-/6ms GET /] nodejs.TypeError: this.logger is not a function
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:9:10)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:36:12

pid: 6549
hostname: C02Y81Q2JG5L

2019-12-16 17:36:44,100 ERROR 6549 [-/::1/-/1ms GET /] nodejs.TypeError: this.logger is not a function
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:9:10)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:36:12

pid: 6549
hostname: C02Y81Q2JG5L

2019-12-16 17:38:01,923 ERROR 6549 [-/::1/-/0ms GET /] nodejs.TypeError: this.logger is not a function
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:9:10)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:36:12

pid: 6549
hostname: C02Y81Q2JG5L

2019-12-16 17:38:03,175 ERROR 6862 [-/::1/-/5ms GET /] nodejs.TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    at JSON.stringify (<anonymous>)
    at respond (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:259:15)
    at handleResponse (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:164:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

pid: 6862
hostname: C02Y81Q2JG5L

2019-12-16 17:38:40,135 ERROR 7018 [-/::1/-/5ms GET /] nodejs.TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    at JSON.stringify (<anonymous>)
    at respond (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:259:15)
    at handleResponse (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:164:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

pid: 7018
hostname: C02Y81Q2JG5L

2019-12-16 17:39:29,567 ERROR 7215 [-/::1/-/5ms GET /] nodejs.TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    at JSON.stringify (<anonymous>)
    at respond (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:259:15)
    at handleResponse (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:164:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

pid: 7215
hostname: C02Y81Q2JG5L

2019-12-16 17:39:30,629 ERROR 7215 [-/::1/-/1ms GET /] nodejs.TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    --> starting at object with constructor 'Pool'
    |     property 'config' -> object with constructor 'PoolConfig'
    |     property 'connectionConfig' -> object with constructor 'ConnectionConfig'
    --- property 'pool' closes the circle
    at JSON.stringify (<anonymous>)
    at respond (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:259:15)
    at handleResponse (/Users/pengzefan/learn/shopping-system/be/node_modules/koa/lib/application.js:164:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

pid: 7215
hostname: C02Y81Q2JG5L

2019-12-16 17:41:31,918 ERROR 7672 [-/::1/-/6ms GET /] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'user' at line 1
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: user
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'user' at line 1"
sqlState: "42000"
index: 0
sql: "user"
name: "ER_PARSE_ERRORError"
pid: 7672
hostname: C02Y81Q2JG5L

2019-12-16 17:42:02,125 ERROR 7802 [-/::1/-/18ms GET /] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'user' at line 1
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: user
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'user' at line 1"
sqlState: "42000"
index: 0
sql: "user"
name: "ER_PARSE_ERRORError"
pid: 7802
hostname: C02Y81Q2JG5L

2019-12-16 17:43:19,392 ERROR 8120 [-/::1/-/4ms GET /] nodejs.TypeError: Cannot read property 'query' of undefined
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:8:36)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:36:12

pid: 8120
hostname: C02Y81Q2JG5L

2019-12-16 17:53:25,560 ERROR 10318 [-/::1/-/12ms GET /] nodejs.SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:11:21)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 10318
hostname: C02Y81Q2JG5L

2019-12-16 17:53:31,418 ERROR 10318 [-/::1/-/5ms GET /] nodejs.SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:11:21)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 10318
hostname: C02Y81Q2JG5L

2019-12-16 18:18:03,597 ERROR 15963 [-/::1/-/5ms GET /] nodejs.TypeError: Cannot read property 'config' of undefined
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/base_context_class.js:30:27)
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg/lib/core/base_context_class.js:13:1)
    at new UserController (/Users/pengzefan/learn/shopping-system/be/app/controller/user.js:5:1)
    at Object.classControllerMiddleware [as getUserList] (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:83:26)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:8:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 15963
hostname: C02Y81Q2JG5L

2019-12-16 18:18:21,788 ERROR 16039 [-/::1/-/5ms GET /] nodejs.TypeError: Cannot read property 'config' of undefined
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/base_context_class.js:30:27)
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg/lib/core/base_context_class.js:13:1)
    at new UserController (/Users/pengzefan/learn/shopping-system/be/app/controller/user.js:5:1)
    at Object.classControllerMiddleware [as getUserList] (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:83:26)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:8:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 16039
hostname: C02Y81Q2JG5L

2019-12-16 18:18:24,922 ERROR 16039 [-/::1/-/0ms GET /] nodejs.TypeError: Cannot read property 'config' of undefined
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/base_context_class.js:30:27)
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg/lib/core/base_context_class.js:13:1)
    at new UserController (/Users/pengzefan/learn/shopping-system/be/app/controller/user.js:5:1)
    at Object.classControllerMiddleware [as getUserList] (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:83:26)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:8:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 16039
hostname: C02Y81Q2JG5L

2019-12-16 18:18:30,341 ERROR 16039 [-/::1/-/1ms GET /] nodejs.TypeError: Cannot read property 'config' of undefined
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/base_context_class.js:30:27)
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg/lib/core/base_context_class.js:13:1)
    at new UserController (/Users/pengzefan/learn/shopping-system/be/app/controller/user.js:5:1)
    at Object.classControllerMiddleware [as getUserList] (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:83:26)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:8:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 16039
hostname: C02Y81Q2JG5L

2019-12-16 18:18:31,247 ERROR 16048 [-/::1/-/4ms GET /] nodejs.TypeError: Cannot read property 'config' of undefined
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/base_context_class.js:30:27)
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg/lib/core/base_context_class.js:13:1)
    at new UserController (/Users/pengzefan/learn/shopping-system/be/app/controller/user.js:5:1)
    at Object.classControllerMiddleware [as getUserList] (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:83:26)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:8:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 16048
hostname: C02Y81Q2JG5L

2019-12-16 18:18:40,768 ERROR 16048 [-/::1/-/1ms GET /] nodejs.TypeError: Cannot read property 'config' of undefined
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/base_context_class.js:30:27)
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg/lib/core/base_context_class.js:13:1)
    at new UserController (/Users/pengzefan/learn/shopping-system/be/app/controller/user.js:5:1)
    at Object.classControllerMiddleware [as getUserList] (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:83:26)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:8:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 16048
hostname: C02Y81Q2JG5L

2019-12-16 18:19:18,466 ERROR 16254 [-/::1/-/5ms GET /] nodejs.TypeError: Cannot read property 'config' of undefined
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/base_context_class.js:30:27)
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg/lib/core/base_context_class.js:13:1)
    at new UserController (/Users/pengzefan/learn/shopping-system/be/app/controller/user.js:5:1)
    at Object.classControllerMiddleware [as getUserList] (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:83:26)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:9:37)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 16254
hostname: C02Y81Q2JG5L

2019-12-16 18:19:26,897 ERROR 16254 [-/::1/-/1ms GET /] nodejs.TypeError: Cannot read property 'config' of undefined
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/base_context_class.js:30:27)
    at new BaseContextClass (/Users/pengzefan/learn/shopping-system/be/node_modules/egg/lib/core/base_context_class.js:13:1)
    at new UserController (/Users/pengzefan/learn/shopping-system/be/app/controller/user.js:5:1)
    at Object.classControllerMiddleware [as getUserList] (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:83:26)
    at HomeController.index (/Users/pengzefan/learn/shopping-system/be/app/controller/home.js:9:37)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 16254
hostname: C02Y81Q2JG5L

2019-12-17 08:07:52,241 ERROR 18256 nodejs.ClusterClientNoResponseError: client no response in 43261238ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 18256
hostname: C02Y81Q2JG5L

2019-12-17 09:30:03,025 ERROR 18256 nodejs.ClusterClientNoResponseError: client no response in 4930471ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 18256
hostname: C02Y81Q2JG5L

2019-12-17 10:52:40,786 ERROR 18256 nodejs.ClusterClientNoResponseError: client no response in 4937664ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 18256
hostname: C02Y81Q2JG5L

2019-12-17 14:22:03,380 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 5408334ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-17 21:19:05,353 ERROR 31398 [-/127.0.0.1/-/17ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 31398
hostname: C02Y81Q2JG5L

2019-12-17 21:19:42,582 ERROR 31411 [-/127.0.0.1/-/12ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 31411
hostname: C02Y81Q2JG5L

2019-12-17 21:20:01,686 ERROR 31416 [-/127.0.0.1/-/14ms POST /register] nodejs.TypeError: Found non-callable @@iterator
    at RegisterService.register (/Users/pengzefan/learn/shopping-system/be/app/service/register.js:8:37)
    at RegisterController.register (/Users/pengzefan/learn/shopping-system/be/app/controller/register.js:7:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 31416
hostname: C02Y81Q2JG5L

2019-12-17 21:20:04,260 ERROR 31416 [-/127.0.0.1/-/1ms POST /register] nodejs.TypeError: Found non-callable @@iterator
    at RegisterService.register (/Users/pengzefan/learn/shopping-system/be/app/service/register.js:8:37)
    at RegisterController.register (/Users/pengzefan/learn/shopping-system/be/app/controller/register.js:7:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 31416
hostname: C02Y81Q2JG5L

2019-12-17 21:20:09,665 ERROR 31416 [-/127.0.0.1/-/1ms POST /register] nodejs.TypeError: Found non-callable @@iterator
    at RegisterService.register (/Users/pengzefan/learn/shopping-system/be/app/service/register.js:8:37)
    at RegisterController.register (/Users/pengzefan/learn/shopping-system/be/app/controller/register.js:7:48)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)
    at next (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:45:18)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-compose/index.js:44:32)

pid: 31416
hostname: C02Y81Q2JG5L

2019-12-17 21:21:02,111 ERROR 31425 [-/127.0.0.1/-/15ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 31425
hostname: C02Y81Q2JG5L

2019-12-18 09:22:50,980 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 43249014ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-18 09:30:04,727 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 433018ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-18 10:49:58,874 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 4794129ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-18 10:51:48,957 ERROR 31425 [-/127.0.0.1/-/118ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 31425
hostname: C02Y81Q2JG5L

2019-12-18 10:53:59,673 ERROR 31883 [-/127.0.0.1/-/19ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 31883
hostname: C02Y81Q2JG5L

2019-12-18 10:58:01,139 ERROR 31900 [-/127.0.0.1/-/32ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 31900
hostname: C02Y81Q2JG5L

2019-12-18 10:59:09,128 ERROR 32134 [-/127.0.0.1/-/13ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 32134
hostname: C02Y81Q2JG5L

2019-12-18 10:59:30,889 ERROR 32146 [-/127.0.0.1/-/13ms POST /register] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''username', 'password') VALUES('pzf', '123')' at line 1
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`('username', 'password') VALUES('pzf', '123')
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''username', 'password') VALUES('pzf', '123')' at line 1"
sqlState: "42000"
index: 0
sql: "INSERT INTO `user`('username', 'password') VALUES('pzf', '123')"
name: "ER_PARSE_ERRORError"
pid: 32146
hostname: C02Y81Q2JG5L

2019-12-18 10:59:45,451 ERROR 32151 [-/127.0.0.1/-/12ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 32151
hostname: C02Y81Q2JG5L

2019-12-18 11:00:20,064 ERROR 32156 [-/127.0.0.1/-/12ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 32156
hostname: C02Y81Q2JG5L

2019-12-18 11:01:01,332 ERROR 32156 [-/127.0.0.1/-/2ms POST /register] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'password' in 'field list'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'password' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_BAD_FIELD_ERRORError"
pid: 32156
hostname: C02Y81Q2JG5L

2019-12-18 11:01:14,585 ERROR 32167 [-/127.0.0.1/-/13ms POST /register] nodejs.ER_WRONG_VALUE_COUNT_ON_ROWError: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user` VALUES('pzf', '123')
code: "ER_WRONG_VALUE_COUNT_ON_ROW"
errno: 1136
sqlMessage: "Column count doesn't match value count at row 1"
sqlState: "21S01"
index: 0
sql: "INSERT INTO `user` VALUES('pzf', '123')"
name: "ER_WRONG_VALUE_COUNT_ON_ROWError"
pid: 32167
hostname: C02Y81Q2JG5L

2019-12-18 11:19:35,850 ERROR 32187 [-/127.0.0.1/-/31ms POST /register] nodejs.ER_NO_REFERENCED_ROW_2Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `id` FOREIGN KEY (`id`) REFERENCES `user` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_REFERENCED_ROW_2"
errno: 1452
sqlMessage: "Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `id` FOREIGN KEY (`id`) REFERENCES `user` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)"
sqlState: "23000"
index: 0
sql: "INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_REFERENCED_ROW_2Error"
pid: 32187
hostname: C02Y81Q2JG5L

2019-12-18 11:25:42,466 ERROR 32187 [-/127.0.0.1/-/3ms POST /register] nodejs.ER_NO_REFERENCED_ROW_2Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_REFERENCED_ROW_2"
errno: 1452
sqlMessage: "Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)"
sqlState: "23000"
index: 0
sql: "INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_REFERENCED_ROW_2Error"
pid: 32187
hostname: C02Y81Q2JG5L

2019-12-18 11:27:52,097 ERROR 32187 [-/127.0.0.1/-/3ms POST /register] nodejs.ER_NO_REFERENCED_ROW_2Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_REFERENCED_ROW_2"
errno: 1452
sqlMessage: "Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)"
sqlState: "23000"
index: 0
sql: "INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_REFERENCED_ROW_2Error"
pid: 32187
hostname: C02Y81Q2JG5L

2019-12-18 11:28:17,118 ERROR 32187 [-/127.0.0.1/-/2ms POST /register] nodejs.ER_NO_REFERENCED_ROW_2Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`id`))
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_REFERENCED_ROW_2"
errno: 1452
sqlMessage: "Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`id`))"
sqlState: "23000"
index: 0
sql: "INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_REFERENCED_ROW_2Error"
pid: 32187
hostname: C02Y81Q2JG5L

2019-12-18 11:34:27,685 ERROR 32187 [-/127.0.0.1/-/1ms POST /register] nodejs.ER_NO_REFERENCED_ROW_2Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`user_id`) ON DELETE CASCADE ON UPDATE CASCADE)
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_REFERENCED_ROW_2"
errno: 1452
sqlMessage: "Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`user_id`) ON DELETE CASCADE ON UPDATE CASCADE)"
sqlState: "23000"
index: 0
sql: "INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_REFERENCED_ROW_2Error"
pid: 32187
hostname: C02Y81Q2JG5L

2019-12-18 11:36:49,195 ERROR 32187 [-/127.0.0.1/-/2ms POST /register] nodejs.ER_NO_REFERENCED_ROW_2Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`user_id`) ON DELETE RESTRICT ON UPDATE RESTRICT)
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_REFERENCED_ROW_2"
errno: 1452
sqlMessage: "Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`user_id`) ON DELETE RESTRICT ON UPDATE RESTRICT)"
sqlState: "23000"
index: 0
sql: "INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_REFERENCED_ROW_2Error"
pid: 32187
hostname: C02Y81Q2JG5L

2019-12-18 11:36:51,321 ERROR 32187 [-/127.0.0.1/-/1ms POST /register] nodejs.ER_NO_REFERENCED_ROW_2Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`user_id`) ON DELETE RESTRICT ON UPDATE RESTRICT)
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_REFERENCED_ROW_2"
errno: 1452
sqlMessage: "Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`user_id`) ON DELETE RESTRICT ON UPDATE RESTRICT)"
sqlState: "23000"
index: 0
sql: "INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_REFERENCED_ROW_2Error"
pid: 32187
hostname: C02Y81Q2JG5L

2019-12-18 11:38:18,699 ERROR 32187 [-/127.0.0.1/-/2ms POST /register] nodejs.ER_NO_REFERENCED_ROW_2Error: ER_NO_REFERENCED_ROW_2: Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`user_id`) ON DELETE CASCADE ON UPDATE CASCADE)
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_REFERENCED_ROW_2"
errno: 1452
sqlMessage: "Cannot add or update a child row: a foreign key constraint fails (`shopping`.`userinfo`, CONSTRAINT `user_id` FOREIGN KEY (`id`) REFERENCES `user` (`user_id`) ON DELETE CASCADE ON UPDATE CASCADE)"
sqlState: "23000"
index: 0
sql: "INSERT INTO `userInfo`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_REFERENCED_ROW_2Error"
pid: 32187
hostname: C02Y81Q2JG5L

2019-12-18 12:21:01,875 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 233939ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-18 13:06:05,730 ERROR 38085 [-/127.0.0.1/-/21ms POST /register] nodejs.ER_NO_DEFAULT_FOR_FIELDError: ER_NO_DEFAULT_FOR_FIELD: Field 'user_id' doesn't have a default value
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')
code: "ER_NO_DEFAULT_FOR_FIELD"
errno: 1364
sqlMessage: "Field 'user_id' doesn't have a default value"
sqlState: "HY000"
index: 0
sql: "INSERT INTO `user`(`username`, `password`) VALUES('pzf', '123')"
name: "ER_NO_DEFAULT_FOR_FIELDError"
pid: 38085
hostname: C02Y81Q2JG5L

2019-12-18 14:29:25,304 ERROR 41192 [-/127.0.0.1/-/4ms POST /login] nodejs.TypeError: Cannot convert undefined or null to object
    at RegisterController.login (/Users/pengzefan/learn/shopping-system/be/app/controller/register.js:21:5)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async /Users/pengzefan/learn/shopping-system/be/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:50
    at async xframe (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-security/lib/middlewares/xframe.js:7:5)
    at async xssProtection (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-security/lib/middlewares/xssProtection.js:7:5)
    at async nosniff (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-security/lib/middlewares/nosniff.js:8:5)
    at async noopen (/Users/pengzefan/learn/shopping-system/be/node_modules/egg-security/lib/middlewares/noopen.js:8:5)
    at async session (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-session/index.js:41:7)
    at async bodyParser (/Users/pengzefan/learn/shopping-system/be/node_modules/koa-bodyparser/index.js:86:5)
    at async /Users/pengzefan/learn/shopping-system/be/node_modules/koa-static-cache/index.js:51:63

pid: 41192
hostname: C02Y81Q2JG5L

